exp/internal/controllers/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package controllers implements experimental controllers.
```

Comment:
@shivanshu1333 Can you please also add a similar doc.go to exp/controllers?

I'm not sure why golangci is not reporting it on the PR, but in two other cases it started reporting it on other PRs after the move PR was merged :/

Reply:
We already have doc.go in exp/controllers, probably that's why golangci is not showing it :)

Reply:
Yup, you're right.

Had a few similar PRs were we didn't have the file and golangci-lint also didn't show it. 

But here you're absolutely right :)